Do not use volumeMode: block in LocalVolume object (#28680)
openshift/openshift-docs: modules/cluster-logging-deploy-cli.adoc

* Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
requires its own storage volume.
+
[NOTE]
====

> If you are using a local volume for persistent storage, do not use a raw block volume (`volumeMode: block` in the `LocalVolume` object) because Elasticsearch cannot use raw block volumes.

Reviewer: ```suggestion
If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: block` in the `LocalVolume` object. Elasticsearch cannot use raw block volumes.
```

> Avoid using parentheses in general text because text in parentheses is decreased in importance.
